tests/test_edge_cases.py

```diff
@@ -509,6 +509,144 @@ class Model(BaseModel):
     }
 
 
+def test_advanced_exclude_nested_lists():
+    class SubSubModel(BaseModel):
+        i: int
+        j: int
+
+    class SubModel(BaseModel):
+        k: int
+        subsubs: List[SubSubModel]
+
+    class Model(BaseModel):
+        subs: List[SubModel]
+
+    m = Model(
+        subs=[
+            SubModel(k=1, subsubs=[SubSubModel(i=1, j=1), SubSubModel(i=2, j=2)]),
```

Comment:
I think it's clear and more consistent with other tests if you use dicts, not models when creating dicst,
```suggestion
            dict(k=1, subsubs=[dict(i=1, j=1), dict(i=2, j=2)]),
```
etc.